samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -198,7 +215,20 @@ CHIP_ERROR BLEConnectivityManager::Init(bt_uuid **serviceUuids, uint8_t serviceU
 	bt_scan_init(&scan_init);
 	bt_scan_cb_register(&scan_cb);
 
-	int err;
+	CHIP_ERROR err = PrepareFiltersForUuid();
+	if (CHIP_NO_ERROR != err) {
+		return err;
+	}
+
+	return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR BLEConnectivityManager::PrepareFiltersForUuid()
```

Comment:
Super minor, but I think it should be `PrepareFilterForUuid` and `PrepareFilterForAddress` (singular instead of plural)